include/fiction/algorithms/simulation/sidb/is_operational.hpp

```diff
@@ -111,6 +134,28 @@ enum class non_operationality_reason : uint8_t
      */
     NONE,
 };
+/**
+ * Reason why the layout is pruned.
+ */
+enum class pruning_reason : uint8_t
```

Comment:
Why would you prune the layout? Prune it of what? This function should only determine operationality, not remove any SiDBs